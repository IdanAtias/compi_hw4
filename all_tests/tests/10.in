F x;
F y;
while(x<y) {
	F z;
	break;
}
F w;

