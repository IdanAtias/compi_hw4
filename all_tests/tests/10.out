-- end scope --
var:z:2:FAHRENHEIT
-- end scope --
var:x:0:FAHRENHEIT
var:y:1:FAHRENHEIT
var:w:2:FAHRENHEIT
